### PR TITLE
Run mypy with torch==1.13.0

### PR DESCRIPTION
**Pull request by carmocca**
## What does this PR do?

See title

### Does your PR introduce any breaking changes? If yes, please list them.

None

### CODE REVIEW

File: .github/workflows/code-checks.yml [hunk 1]
```diff
@@ -45,10 +47,10 @@ jobs:
 
     - name: Install dependencies
       run: |
-        pip install mypy==0.982  # TODO: pull this verion deom requirements/test.txt
-        pip install torch==1.12 --find-links https://download.pytorch.org/whl/cpu/torch_stable.html
-        python ./requirements/pytorch/adjust-versions.py requirements/pytorch/extra.txt
-        # todo: adjust requirements for both code-bases
+        pip install mypy==0.982
```
Comment (Borda) [inline .github/workflows/code-checks.yml:50]: shall this be part of the requirements so it is managed by dependabot?

Comment (carmocca) [reply .github/workflows/code-checks.yml:50]: I suggest not. It requires manual fixes for basically all releases

Comment (Borda) [reply .github/workflows/code-checks.yml:50]: yes, but this is still under PR so no merge until it is resolved

> for basically all releases

not sure what releases are you talking about... :otter:

Comment (carmocca) [reply .github/workflows/code-checks.yml:50]: mypy releases

Comment (Borda) [reply .github/workflows/code-checks.yml:50]: we can skip, but at least we wont be frozen for ever :)

**Pull request closed without merging**